test/functional/services/saved_query_management_component.ts

```diff
@@ -166,8 +166,9 @@ export class SavedQueryManagementComponentService extends FtrService {
     const isOpenAlready = await this.testSubjects.exists('saved-query-management-popover');
     if (isOpenAlready) return;
 
+    await this.testSubjects.click('saved-query-management-popover-button');
+
     await this.retry.waitFor('saved query management popover to have any text', async () => {
-      await this.testSubjects.click('saved-query-management-popover-button');
       const queryText = await this.testSubjects.getVisibleText('saved-query-management-popover');
```

Comment:
What happened here `saved-query-management-popover-button` was clicked multiple times, because the visible text was not yet rendered. When it was clicked again the popover management was closed ... so the visible text was again empty